test/acqusition.test.mjs

```diff
@@ -0,0 +1,118 @@
+    { input: 'newsletter', expected: 'owned:email' },
+    { input: 'instagram', expected: ':social:instagram' },
+    { input: 'dv360', expected: '' },
+    { input: 'email', expected: 'owned:email' },
+    { input: 'programmatic', expected: 'paid:display' },
+    { input: 'googleads', expected: 'paid:search:google', only: true },
+    { input: 'sfmc', expected: '' },
+    { input: 'google-ads', expected: 'paid:search:google', only: true },
+    { input: 'an', expected: '' },
+    { input: 'marketo', expected: 'owned:email:marketo', only: true },
+    { input: 'hs_email', expected: 'owned:email' },
+    { input: 'social', expected: ':social', only: true },
+    { input: 'microsoft', expected: 'paid:display:microsoft', only: true },
+    { input: 'adwords', expected: 'paid:search:google', only: true },
+    { input: 'pinterest', expected: ':social:pinterest' },
+    { input: 'gdn', expected: '' },
+    { input: 'facebook-instagram', expected: ':social:instagram', only: true },
+    { input: 'twitter', expected: ':social:x' },
+    { input: 'snapchat', expected: ':social:snapchat' },
+    { input: 'eloqua', expected: 'owned:email:eloqua', only: true },
+    { input: 'yahoo', expected: 'paid:search:yahoo', only: true },
+    { input: 'teads', expected: 'paid::teads' },
+    { input: 'criteo', expected: 'paid:display:criteo', only: true },
+    { input: 'acs', expected: '' },
+    { input: 'taboola', expected: 'paid:display:taboola', only: true },
+    { input: 'kaufland marketing', expected: '' },
+    { input: 'search', expected: 'paid:search', only: true },
+    { input: 'abandoned-cart', expected: '' },
+    { input: 'zalo', expected: '' },
+    { input: 'website', expected: 'owned:web' },
+    { input: 'outbrain', expected: 'paid:display:outbrain', only: true },
+    { input: 'google_pmax', expected: 'paid:search:google', only: true },
+    { input: 'substack', expected: 'owned:email:substack', only: true },
+    { input: 'line', expected: ':social:line', only: true },
+    { input: 'spotify', expected: 'paid:display:spotify', only: true },
+    { input: 'display', expected: 'paid:display' },
+    { input: 'google_deman', expected: 'paid:search:google', only: true },
+    { input: 'ttd', expected: '' },
+    { input: 'sms', expected: 'owned:sms' },
+    { input: 'qr', expected: 'owned:qr' },
+    { input: 'reddit', expected: 'paid:social:reddit', only: true },
+    { input: 'dbm', expected: '' },
+    { input: 'google_search', expected: 'paid:search:google', only: true },
+    { input: 'qrcode', expected: 'owned:qr' },
+    { input: 'linkin.bio', expected: 'owned:social', only: true },
+    { input: 'cpc', expected: 'paid' },
+    { input: 'paid', expected: 'paid' },
+    { input: 'email', expected: 'owned:email' },
+    { input: 'social', expected: ':social', only: true },
+    { input: 'yext', expected: 'paid:local:yext', only: true },
+    { input: 'video', expected: ':video', only: true },
+    { input: 'referral', expected: '' },
```

Comment:
I think we are missing a whole top-level category next to owned and paid. Referrals are essentially neither owned, nor paid, they are just hypertext links

Reply:
I remember reading that GA was considering anything that is not paid, organic and direct a referral… it was essentially their fallback